### ISSUE TITLE
Backfilling data to continuous aggregate

### ISSUE THREAD

**Issue by ConradKurth** [timescale/timescaledb#3546]
**Relevant system information:**
 - OS: [e.g. Ubuntu 16.04, Windows 10 x64, etc]
 - PostgreSQL version (output of `postgres --version`): [e.g. 13.2, 12.0]
 - TimescaleDB version (output of `\dx` in `psql`): [e.g. 1.0.0]
 - Installation method: [e.g., "using Docker", "apt install", "source"]

Running on timescale cloud. Postgres v12 and latest version of timescale as of writting

**Describe the bug**

Unexpected behavior for doing a backfill. I am wanting to insert data that exceeds my hypertable time window. 

I have some backfilled data that i want to insert for the past 2 years. I can inject this data into the hypertable and then call refresh continuous aggregate, but the aggregate just refreshes based off what is in the hypertable. I would want it to keep all existing data in the aggregate but just add in the new data I want backfilled.

**To Reproduce**
```
CREATE TEMPORARY TABLE  temp  AS SELECT * FROM some_table WITH NO DATA;

INSERT INTO temp(column_a,value,time) VALUEs,,, <some values here dating back>


CALL decompress_backfill(staging_table=>'temp', destination_hypertable=>'some_table');

CALL refresh_continuous_aggregate('wine_price_summary_daily',NULL,NULL);
DROP TABLE IF EXISTS temp
```
**Expected behavior**

Refer to the bug issue. And no, setting bounds is not the root of this problem because I want to be able to backfill data way past the 90 days table temp keeps data around for

**Actual behavior**
A clear and concise description of what _actually_ happened.

All of the data past 90 days from the continuous aggregate is dropped

**Screenshots**

**Additional context**

**Comment by gayyappan** [comment]
```CALL refresh_continuous_aggregate('some_table_summary',NULL,NULL);```
The semantics is to refresh everything from beginning to the end since you don't specify a specific start/end point.
.
If you want to stop the refresh at a certain date , you have to specify that in the refresh.
``` CALL refresh_continuous_aggregate(<cont agg name>, NULL, '2019-01-01'); ```

**Comment by ConradKurth** [comment]
Yes, we are essentially inserting data from start to end. I understand  that I can specify stop and start endpoints but I am refreshing the entire aggregate. 

I want to maintain all the data already existing in the continuous aggregate as well, which per a slack conversation does not exist.

**Comment by gayyappan** [comment]
Could you elaborate why something like this does not work?
```
SELECT now() - '90 days'::interval;
CALL refresh_continuous_aggregate( <cont_agg_name>, NULL, <date from above> )
```

This refreshes the aggregate from beginning  to the <date specified > . It does not touch the 90 days worth of data.

**Comment by ConradKurth** [comment]
Option 2. I may not have made this clear I realize. The hypertable drops data after 90s days. As for your further comment, we do still need to keep the aggregated data n the continuous aggregate

**Comment by davidkohn88** [comment]
I think what you're looking for is something like a `refresh_no_delete` where it would only add data to the continuous aggregate.

For now, a workaround would be a query that essentially does that manually. Usually when we do a refresh, we do a delete in teh time period and then a re-populate. As long as the data you've inserted is disjoint, you should be able to just do the insert (if it's not disjoint, it gets weird and hard). 

So you'd go to the `_timescaledb_catalog.continuous_agg` table and look up the `partial_view_schema`,`partial_view_name` and then the `timescaledb_information.continuous_aggregates` view to get the `materialization_hyperable_schema` and `materialization_hypertable_name`. You'll also need to know the time column in the materialized view, which should have the same name in the partial view.  Then you'd do:

```SQL
INSERT INTO materialization_hypertable_schema.materialization_hypertable_name SELECT * FROM partial_view_schema.partial_view_name WHERE time_column < now() - '90 days' 
```
 Then I'd refresh the view over the past 90 days (maybe in batches).
